execution-engine/src/fs.rs

```diff
@@ -644,6 +644,17 @@ impl FileSystem {
     /// A rust-style base implementation for `fd_write`. It directly calls `fd_pwrite` with the
     /// current `offset` of Fd `fd` and then calls `fd_seek`.
     pub(crate) fn fd_write(&mut self, fd: Fd, buf: &[u8]) -> FileSystemResult<Size> {
+        // Redirect writes to fd(0) and fd(1) to the host's stdout and stderr respectively
+        if fd.0 == 0 || fd.0 == 1 {
```

Comment:
I think we should introduce explicit constants for the `stdout`, `stdin`, and `stderr` handles, rather than comparing against bare numeric values.  Can you do that?

Reply:
You might want to move these changes to `fd_pwrite` function. Another solution is to read the content in `Fd(1)` and `Fd(2)`, namely `stdout` and `stderr`, and dump them to the host, meaning cross the TEE boundary. You can define a function like `read_stdout` and `read_stderr` to return the content of these two files and let other components, e.g. veracruz runtime, decide if they want to call these functions and get the debug result.